README.md

```diff
@@ -48,12 +42,59 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 For more details and latest updates, see [TiDB docs](https://docs.pingcap.com/tidb/stable) and [release notes](https://docs.pingcap.com/tidb/dev/release-notes).
 
+## Community
+
+You can join these groups and chats to discuss and ask TiDB related questions:
+
+- [Contributors Mailing list](https://lists.tidb.io/g/contributors)
+- [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)
```

Comment:
Can we change this to [Slack Channel (in Chinese and English)](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)?

Reply:
Why do you want to add this message? Generally I hope our channel in English with optional Chinese only channel. This is the default understanding and we don't have to mention it intentionally.

Reply:
Yes. You are right.